StripePaymentSheet/StripePaymentSheet/Source/Categories/String+Localized.swift

```diff
@@ -311,4 +311,18 @@ extension String.Localized {
             "Title shown above a view allowing the customer to save a card."
         )
     }
+
+    static var after_pay_subtitle_text: String {
+        STPLocalizedString(
+            "%d interest-free payments of %@",
```

Comment:
Multiple parameters in a localized string need to include the position of the arguments, it should be like "%1$d interest-free payments of %2$d" so that languages can change the order of the parameters in their translation